enhancements/builds/entitlements.md

```diff
@@ -0,0 +1,295 @@
+---
+title: openshift-builds-improve-use-of-rhel-entitlements
+
+authors:
+  - "@gabemontero"
+  - "@bparees" (provided a lot of input in tracking Jira)
+
+reviewers:
+  - "@bparees"
+  - "@adambkaplan"
+  - "@who else?"
+  
+approvers:
+  - "@bparees"
+  - "@adambkaplan"
+  - "@who else?"
```

Comment:
@mrunalp should agree to any plan here as well.

Reply:
he'll be in there in the next push thanks